.circleci/main.yml

```diff
@@ -1414,15 +1417,25 @@ jobs:
       - run: #Note: --no-ansi https://github.com/python-poetry/poetry/issues/7184
           name: Install Python Dependencies with Poetry
           command: poetry --no-ansi install
-      - install-python-test-deps
-
-      # export PACH_PYTHON_ENTERPRISE_CODE=${ENT_ACT_CODE}
-      # export export PACH_PYTHON_AUTH_TOKEN=iamroot
       - run:
-          name: Run Tests
+          name: Start minikube
+          command: minikube start --vm-driver=docker --kubernetes-version=v1.19.0 --cpus=7 --memory=12g --wait=all
+      - run:
+          name: Install pach
           command: |
-            nohup pachctl port-forward &
-            poetry run pytest -vvv tests
+            helm install pachd ../etc/helm/pachyderm --set deployTarget=LOCAL,pachd.image.tag=${CIRCLE_SHA1},proxy.service.type=LoadBalancer
+            kubectl create namespace enterprise --dry-run=client -o yaml | kubectl apply -f -
+            helm install pachd-enterprise ../etc/helm/pachyderm --namespace enterprise --set deployTarget=LOCAL,pachd.image.tag=${CIRCLE_SHA1},proxy.service.type=LoadBalancer,proxy.service.httpPort=81,pachd.enterpriseLicenseKey=${ENT_ACT_CODE}
+      - run:
+          name: Wait for pach # need to wait before testing kubectl as pach is not visible to kubectl yet
+          command: |
+            sleep 20
+            kubectl wait -A --for=condition=ready pod -l app=pachd --timeout=5m
+            echo "PACH_PYTHON_TEST_ADDRESS=$(minikube service pachyderm-proxy --url)" >> "$BASH_ENV"
+            echo "PACH_PYTHON_TEST_ADDRESS_ENTERPRISE=$(minikube --namespace enterprise service pachyderm-proxy --url)" >> "$BASH_ENV"
+      - run:
+          name: Run Tests
+          command: PACH_PYTHON_TEST_ADDRESS="$(minikube service pachyderm-proxy --url)" PACH_PYTHON_TEST_ADDRESS_ENTERPRISE="$(minikube --namespace enterprise service pachyderm-proxy --url)" poetry run pytest -vvv tests
```

Comment:
Could do:
```
command: |
  PACH_PYTHON_TEST_ADDRESS="$(minikube service pachyderm-proxy --url)" 
  PACH_PYTHON_TEST_ADDRESS_ENTERPRISE="$(minikube --namespace enterprise service pachyderm-proxy --url)" 
  poetry run pytest -vvv tests

Reply:
You may need to wait for pachyderm to be ready by trying to hit it with an inspect or something repeatedly